src/App.js

```diff
@@ -1,11 +1,15 @@
 import './App.css';
-import Navbar1 from "./NavBar1"
-
+import Navbar1 from "./components/NavBar1"
+import LandingPage from "./components/LandingPage"
+import AboutMe from "./components/AboutMe"
 
 function App() {
   return (
     <div className="App">
+      
+      <LandingPage></LandingPage>
```

Comment:
This is your landing page? it's just a gif

Reply:
yeah, I'm planning to add white font text. Should look amazing with the color contrast.